app/controllers/shops_controller.rb

```diff
@@ -2,6 +2,10 @@ class ShopsController < ApplicationController
   before_action :set_user_shop, only: [:edit, :update, :destroy]
   before_action :authenticate_user!, except: [:index, :show]
 
+  add_breadcrumb "Shops", :shops_path
+
+  add_breadcrumb "New Shop", :new_shop_path
```

Comment:
Go to a single shop page and you should be getting a similar breadcrumb: Shops > New Shop > My Cool Shop.